src/packaging/docker-compose.yml

```diff
@@ -2,50 +2,92 @@ version: '2.1'
 
 services:
   cassandra:
-    image: cassandra:3.11
+    extends:
+      file: ./common-services.yml
+      service: cassandra-common
+
+  cassandra-ssl:
+    extends:
+      file: ./common-services.yml
+      service: cassandra-common
+    entrypoint: /docker-alt-entrypoint.sh cassandra -f
     env_file:
-      - ./docker-services/cassandra/cassandra.env
-    mem_limit: 4g
-    memswap_limit: 4g
-    mem_swappiness: 0
-    ports:
-      - "7000:7000"
-      - "7001:7001"
-      - "7199:7199"
-      - "9042:9042"
+      - ./docker-services/cassandra-ssl/cassandra-ssl.env
     volumes:
-      - ./data/cassandra:/var/lib/cassandra
-      - ./docker-services/cassandra/jmxremote.access:/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/management/jmxremote.access
-      - ./docker-services/cassandra/jmxremote.password:/etc/cassandra/jmxremote.password
+      - ./ssl-stores/cassandra-server-keystore.jks:/etc/ssl/cassandra-server-keystore.jks
+      - ./ssl-stores/generic-server-truststore.jks:/etc/ssl/generic-server-truststore.jks
+      - ./docker-services/cassandra-ssl/docker-entrypoint.sh:/docker-alt-entrypoint.sh
 
   cqlsh:
```

Comment:
True. Added one in.

What I have done is added in a `cqlsh-common` service into _common-services.yml_. I then derive the `cqlsh`, `cqlsh-ssl`, `cqlsh-initialize-reaper_db`, and `cqlsh-initialize-reaper_db-ssl` services from that. Note that the `cqlsh-initialize-reaper_db`, and `cqlsh-initialize-reaper_db-ssl` replace the `initialize-reaper_db` and `initialize-reaper_db-ssl` services respectively.

The reason I did this was to avoid duplicating the CQLSH SSL connection process.

@joaquincasares  - let me know what you think.

Reply:
Oh, that was a clever reuse. Looks good! :)